### PR TITLE
Add build/start scripts for production + documentation

### PR DESCRIPTION
**Pull request by victorb**
adds build-prod.sh which builds images and pushes them to quay and start-prod.sh that fetches latest changes and possibly stops/starts container if image changed.

### CODE REVIEW

**Comment by victorb** [comment]
Please review! Should work. Basically added two scripts, one for building image and pushing it. One for running the image on the host. Is relatively straightforward.

File: start-prod.sh [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+CURRENT_IMAGE=$(docker inspect jenkins -f "{{ .Config.Image }}" || echo)
+if [ "$IMAGE_TO_DEPLOY" == "$CURRENT_IMAGE" ]; then
+	echo "Image up-to-date"
+else
+	echo "Different image currently running, deploying new image"
+	docker stop jenkins || true
+	docker rm jenkins || true
+	docker run -d \
+		--name jenkins \
+		-p 80:8080 \
```
Comment (victorb) [inline start-prod.sh:40]: This should change in the future once we have nginx or something in front of it. I think for now it's Good Enough™

File: README.md [hunk 2]
```diff
@@ -22,6 +22,20 @@ username: admin
 password: admin
 ```
 
+## Production Deploy
+
+- Run `./build-prod.sh` on machine with access to our Quay registry
+- Run `./provsn jenkins run './start-prod.sh $VERSION'` from ipfs/infrastructure
```
Comment (ghost) [inline README.md]: provsn doesn't have a jenkins command, do you mean something like `./provsn exec jenkins '/path/to/start-prod.sh $VERSION'`?

**Comment by ghost** [comment]
LGTM :+1:

**Review by ghost — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (victorb) [reply README.md]: Ah, yes. Correcting.

**Pull request merged**